ReactSkia/BUILD.gn

```diff
@@ -60,6 +61,8 @@ source_set("ReactSkia") {
     "RNInstance.h",
     "sdk/NotificationCenter.cpp",
     "sdk/NotificationCenter.h",
+    "sdk/TVNavigationEventEmitter.cpp",
```

Comment:
The filename should be RSkTVNavigationEventEmitter.cpp and RSkTVNavigationEventEmitter.h and should be under "ReactSkia/modules" folder